site/docs/usage/web-ui.md

```
@@ -1,58 +1,145 @@
+
+## URL Parameters
+
+Viewer state syncs to the URL—bookmark or share filtered views:
 
-See [sharing documentation](/docs/usage/sharing) for authentication, enterprise options, and CI/CD setup.
+| Parameter | Values |
+|-----------|--------|
+| `filterMode` | `all`, `failures`, `passes`, `errors`, `different`, `highlights` |
+| `search` | Any text |
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)